src/pages/postgraphile/smart-tags.md

```diff
@@ -325,9 +325,16 @@ Since version
 
 Applies to:
 
-- Functions returning `SETOF`: adds the `orderBy` argument to this field
-- Computed column functions with no required parameters: allows this function to
-  be used as an order in the `orderBy` argument for the parent table
+- Functions returning `SETOF`: adds the `orderBy` argument to this connection,
+  allowing to _sort_ the set by any of its scalar fields.
```

Comment:
```suggestion
  allowing to _sort_ the set by its columns/attributes.
```

I'd rather not say "any" here, as things like `ignoreIndexes: false` may remove certain ones.